extensions/kubernetes-client/runtime-internal/src/main/java/io/quarkus/kubernetes/client/runtime/KubernetesClientUtils.java

```diff
@@ -42,6 +42,7 @@ public static Config createConfig(KubernetesClientBuildConfig buildConfig, TlsCo
                 .withProxyUsername(buildConfig.proxyUsername.orElse(base.getProxyUsername()))
                 .withProxyPassword(buildConfig.proxyPassword.orElse(base.getProxyPassword()))
                 .withNoProxy(buildConfig.noProxy.orElse(base.getNoProxy()))
+                .withHttp2Disable(base.isHttp2Disable())
```

Comment:
What's the rationale for this?

Reply:
This is a configuration set in:

https://github.com/quarkusio/quarkus/blob/4c6317a4caab98aa4d02e8ec8b32744d7a3c8594/test-framework/kubernetes-client/src/main/java/io/quarkus/test/kubernetes/client/AbstractKubernetesTestResource.java#L21

But it's ignored later on. With this change, the configuration is propagated to the final applicable Config.

 I guess that no one is using `https` mock server mode with JDK 8, since it won't work. Since for the OpenShift version there's no option to run https-less (until the fabric8 bugs gets fixed), I faced this issue from the first minute.

Reply:
Understood, thanks